src/authorizations/users.rs

```diff
@@ -1,26 +1,39 @@
+use crate::config::Config;
+
 fn authorized_users() -> Vec<String> {
-    dotenv::var("AUTHORIZED_USERS")
-        .unwrap()
-        .split(',')
-        .map(String::from)
-        .collect()
+    let bot_config = Config::default();
```

Comment:
I prefer to generate the configuration once in `main()` and then pass it around to whereever it is needed. Doing this reduces the dependency on global state, and makes testing a lot easier in my experience. Many functions currently depending on environment variables will become pure functions this way.